_sass/minima/_base.scss

```diff
@@ -241,6 +241,9 @@ pre {
   width: 16px;
   height: 16px;
   display: inline-block;
+  /* This is needed just for google scholar. */
+  filter: grayscale(100%);
```

Comment:
For some reason google scholar needed this to stay gray like all the other links. Were the other links just gray by default? I don't really know web dev so it was hard for me to trace where currentColor was coming from.